### PR TITLE
feat: add proposals test

### PR DESCRIPTION
**Pull request by taro04**
wip

### CODE REVIEW

File: projects/portal/src/app/pages/vote/proposals/proposal/proposal.usecase.service.ts [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    return proposalID$.pipe(mergeMap((id) => this.cosmosRest.getProposal$(id)));
+  }
+  proposalType$(
+    proposal$: Observable<InlineResponse20027Proposals | undefined>,
+  ): Observable<string | undefined> {
+    return proposal$.pipe(
+      map((proposal) => {
+        //console.log('prop_debug', proposal);
+        if (proposal && proposal.content) {
+          return (proposal.content as any)['@type'];
```
Comment (taro04) [inline projects/portal/src/app/pages/vote/proposals/proposal/proposal.usecase.service.ts:34]: @yutaro-elk 
How do I make a mock of the object that have '@type' property?

Comment (yutaro-elk) [reply projects/portal/src/app/pages/vote/proposals/proposal/proposal.usecase.service.ts:34]: It's okay if you surround it with "

```ts
      const mockContent: InlineResponse200Accounts = {
        "@type": ""
        type_url: 'test_url',
        value: 'test_value',
      };
```

**Comment by taro04** [comment]
@yutaro-elk 

refactored as below
・Delete local variables of usecase service
・Removed the code in the usecase service constructor.

please, review. 🙏

**Review by yutaro-elk — APPROVED** [review]
LGTM!

**Pull request merged**